analysis/bin/singletopAnalyser.cc

```diff
@@ -1,10 +1,13 @@
 //#define nanoAnalyser_cxx
 #define Events_cxx
 #include "DataFormats/HepMCCandidate/interface/GenStatusFlags.h"
-#include "nano/analysis/interface/nanoBase.h"
+//#include "nano/analysis/interface/nanoBase.h"
```

Comment:
Did you mean to commit the singleTop analyser? shouldn't BH be doing this instead?

Reply:
hmm... actually I have no memory about that...I merged with BH's before, and It could be what he intended.